src/atomate2/common/schemas/phonons.py

```diff
@@ -406,6 +406,16 @@ def from_forces_born(
             kppa=kpoint_density_dos,
             force_gamma=True,
         )
+        # projected dos 
+        if kwargs.get("calculate_pdos", False):
+            phonon.run_mesh(kpoint.kpts[0], with_eigenvectors=True, is_mesh_symmetry=False)
+            phonon_dos_sigma = kwargs.get("phonon_dos_sigma", None)
+            dos_use_tetrahedron_method = kwargs.get("dos_use_tetrahedron_method", True)
+            phonon.run_projected_dos(
+                sigma=phonon_dos_sigma, use_tetrahedron_method=dos_use_tetrahedron_method
+            )
+            phonon.write_projected_dos()
+
```

Comment:
Does it make sense to also add an automatic plot here? Likely a bit more work. I am fine with just adding this for now.

Reply:
This directly relates to the next question: do you want the option to save it in the schema? If so, you would need to develop a parser.

Reply:
> Does it make sense to also add an automatic plot here? Likely a bit more work. I am fine with just adding this for now.

I think it could be useful to provide summed PDOS, for example by element type or symmetry-equivalent atoms. However, user needs in this area are often quite diverse, making it challenging to implement something that meets all needs as straightforwardly as the TDOS plot.


> This directly relates to the next question: do you want the option to save it in the schema? If so, you would need to develop a parser.

The current approach saves the PDOS in an external file, similar to the TDOS format. I believe a simple adjustment to the default filename to match the style of atomate2 (e.g., `phonon_pdos.yaml` instead of the default `projected_dos.dat` in phonopy) would suffice.

Reply:
I am fine with this for now. But is it really a yaml? If not, rather keep the dat

Reply:
Actually, it is not a yaml; it was just renamed with a `.yaml` extension in atomate2 for TDOS.